### PR TITLE
[zh] Synchronize the README file

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
Weird. Let me try rebase.

File: README-zh.md [hunk 1]
```diff
@@ -1,182 +1,144 @@
 # Kubernetes 文档
+
```
Comment (tanjunchen) [inline README-zh.md:2]: /lgtm

**Comment by tengqm** [comment]
@jimangel Looks like changes to README requires a higher privilege to approve.

**Pull request merged**